e2e/cypress/test_module/cypress/features/dashboard-page/new-product/new-product.feature

```diff
@@ -0,0 +1,26 @@
+Feature: | New Product |
+
+  Background: Before each
+    Given I authenticate to the system under "ROLE_MANAGER" role
+    When I click on the Dashboard button
+    When I click on the 'products' tab
+
+  Scenario Outline: Create a new product
+    When I click on the 'New product' button
+    Then I should see the 'Create a product' form
+    When I fill in the 'Image URL' field with "<imageUrl>"
+    And I fill in the 'Product name' "en" field with "<productNameEn>"
+    And I fill in the 'Product name' "uk" field with "<productNameUa>"
+    And I fill in the 'Product description' "en" field with "<productDescriptionEn>"
+    And I fill in the 'Product description' "uk" field with "<productDescriptionUa>"
+    And I fill in the 'Price' field with "<price>"
+    And I fill in the 'Quantity' field with "<quantity>"
+    And I select 'Category' as "<category>"
+    And I check the 'Visible to customers' checkbox
+    When I click on the 'Create product' button
+    Then I should see snackbar with message 'Product successfully created'
+
+    Examples:
+      | imageUrl                      | productNameEn | productNameUa    | productDescriptionEn                                  | productDescriptionUa                        | price | quantity | category |
```

Comment:
It looks like smth heppend with formatting

Reply:
Locally it will be okay. It's because github has some formatting restrictions I guess